support-frontend/assets/helpers/productCatalogue.ts

```diff
@@ -0,0 +1,288 @@
+import { newspaperCountries } from 'helpers/internationalisation/country';
+import { gwDeliverableCountries } from 'helpers/internationalisation/gwDeliverableCountries';
+
+export const productCatalog = window.guardian.productCatalog;
+
+export const productCatalogDescription = {
+	SupporterPlusWithGuardianWeekly: {
+		label: 'Digital + print',
+		benefitsSummary: [
+			'The rewards from ',
+			{ strong: true, copy: 'All-access digital' },
+		],
+		benefits: [
+			{
+				copy: 'Guardian Weekly print magazine delivered to your door every week  ',
+				tooltip: `Guardian Weekly is a beautifully concise magazine featuring a handpicked selection of in-depth articles, global news, long reads, opinion and more. Delivered to you every week, wherever you are in the world.`,
+			},
+		],
+	},
+	DigitalSubscription: {
+		label: 'The Guardian Digital Edition',
+		benefits: [
+			{
+				copy: 'The Editions app. Enjoy the Guardian and Observer newspaper, reimagined for mobile and tablet',
+			},
+			{ copy: 'Full access to our news app. Read our reporting on the go' },
+			{ copy: 'Ad-free reading. Avoid ads on all your devices' },
+			{
+				copy: 'Free 14 day trial. Enjoy a free trial of your subscription, before you pay',
+			},
+		],
+	},
+	NationalDelivery: {
+		label: 'National Delivery',
+		delivery: true,
+		addressCountries: newspaperCountries,
+	},
+	SupporterPlus: {
+		label: 'All-access digital',
+		benefits: [
+			{
+				copy: 'Unlimited access to the Guardian app',
+				tooltip: `Read beyond our 20 article-per-month limit, enjoy offline access and personalised recommendations, and access our full archive of journalism. Never miss a story with the Guardian News app – a beautiful, intuitive reading experience.`,
+			},
+			{ copy: 'Ad-free reading on all your devices' },
+			{
+				copy: 'Exclusive newsletter for supporters, sent every week from the Guardian newsroom',
+			},
+			{
+				copy: 'Far fewer asks for support',
+				tooltip: `You'll see far fewer financial support asks at the bottom of articles or in pop-up banners.`,
+			},
+		],
+	},
+	GuardianWeeklyRestOfWorld: {
+		label: 'The Guardian Weekly',
+		delivery: true,
+		addressCountries: gwDeliverableCountries,
+	},
+	GuardianWeeklyDomestic: {
+		label: 'The Guardian Weekly',
+		delivery: true,
+		addressCountries: gwDeliverableCountries,
+	},
+	SubscriptionCard: {
+		label: 'Newspaper subscription',
+		delivery: true,
+	},
+	Contribution: {
+		label: 'Support',
+		benefits: [
+			{
+				copy: 'Exclusive newsletter for supporters, sent every week from the Guardian newsroom',
+			},
+		],
+	},
+	HomeDelivery: {
+		label: 'Home Delivery',
+		delivery: true,
+		addressCountries: newspaperCountries,
+	},
+};
+
+/** These `ratePlans` will eventually becomes part of the SupportPlus product in Zuora */
+export const supporterPlusWithGuardianWeekly = {
```

Comment:
@rupertbates might be of interest to you - basically I am hard-coding what we would expect to get from Zuora for now so we can remove the middle-types we've introduced to support this.